### ISSUE TITLE
Not really remove the file and records

### ISSUE THREAD

**Issue by loongmxbt** [lc3t35/orion-filesystem-local#4]
Files.remove(cfs_id) can find the record in FS.collection and actually remove file on server.

file._id is the orionFiles id, Files.remove(file._id) only does a fake remove action in orion, but the records in cfs and the file on server still exsit.

**Comment by gam-ragnar** [comment]
Does this require a fix in the orion filesystem package?

**Comment by loongmxbt** [comment]
Not really, it's just collection FS. To actually remove files is what the provider code does.
I have fixed the problem in https://github.com/loongmxbt/orion-lfs . 
It takes me days because the cfs docs do not have the remove example and lack details.
I tried remove FS.file and many other things. Finally the cfs_id works.
I would very much like to P/R this repo. But I'll go to sleep. Hope someone still online to fix this!

Notice: In upload, the `file` is not the same one as in delete.

```
var meta = {
        cfs_id : file._id
      };
      success(fileUrl, meta);
```

```
var cfs_id = file.meta.cfs_id;

    // remove record in cfs and the related file in directory
    // Here the Files.remove() requires cfs_id, not a FS.file
    Files.remove(cfs_id, function(err, file) {
      if (err) {
        console.log('error', err);
      } else {
        console.log('remove success');
        success();  // remove record in orion
      };
    });
```

**Comment by gam-ragnar** [comment]
Good work get some sleep we finalize when you are up
